examples/perm_dc_omega.py

```diff
@@ -1,29 +1,48 @@
 from examples.agents.simple_controllers import Controller
 import time
 import sys, os
+
 sys.path.append(os.path.abspath(os.path.join('..')))
 import gym_electric_motor as gem
 from gym_electric_motor import reference_generators as rg
 from gym_electric_motor.visualization import MotorDashboard
 
+"""
+Run this file from within the 'examples' folder:
+>> cd examples
+>> python perm_dc_omega.py
+
+Description:
+        Environment to control a Continuously controlled DC Permanently Excited Motor.
+
+        Controlled Quantity: 'omega'
+
+        Limitations: Physical limitations of the motor like Current,Speed
+
+        Converter : FourQuadrantConverter from converters.py
+
+"""
+
 if __name__ == '__main__':
+
+    """Discrete mode: It is the mode where the input command causes the controlled variable to change its 
```

Comment:
The explanations to both, the continuous as well as the discrete action mode are not correct:

Continuous mode: action is the average (normalized) voltage per time step which is assumed to be transferred to a PWM unit to generate the switching sequence (not part of the model, since we are working with averaged quantities). Hence, the action a continuous quantity.

Discrete mode: the action is the switching state  of the power converter i.e. a quantity from a discrete set of switching states which can be generated by the converter. Hence we directly generate the switching sequence by this control approach which default step time is therefore much smaller compared to the continuous time framework.